stories/offClickWrapper/offClickWrapperExample.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { OffClickWrapper, Container } from 'src/shared-components';
+
+class OffClickWrapperExample extends React.Component {
+  state = {
+    text: '   Click outside this container to call the onOffClick handler...',
```

Comment:
Not that important, but was there a reason for the spaces before "Click"?

Reply:
must have been a copy paste from the doc, I changed that line a few times during development. Will fix and push and merge